distributed/worker_client.py

```diff
@@ -9,10 +9,10 @@
 from dask.base import tokenize
 from tornado import gen
 
-from .client import Client, Future, pack_data, unpack_remotedata
+from .client import AllExit, Client, Future, pack_data, unpack_remotedata
 from .sizeof import sizeof
 from .threadpoolexecutor import secede
-from .utils import log_errors, sync, tokey, ignoring
+from .utils import All, log_errors, sync, tokey, ignoring
```

Comment:
Note this was a bug that was hidden by the overly broad `ignoring(Exception)` in `WorkerClient._gather`.